dll/shellext/zipfldr/zipfldr.cpp

```diff
@@ -109,9 +109,13 @@ BOOL WINAPI
 RouteTheCall(
     IN HWND hWndOwner,
     IN HINSTANCE hInstance,
-    IN LPWSTR lpNamedPipeName,
+    IN LPCSTR lpStringArg,
     IN INT Show)
 {
-    UNIMPLEMENTED;
-    return FALSE;
+    CStringW path = lpStringArg;
```

Comment:
Does it automatically perform the ANSI to UNICODE conversion?

Reply:
Yeap.

Reply:
This is one of the reasons why `CString` was developed further, because it makes life easier.